From ce2628da1d5042502a943afe57c3553b60aaefe1 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Fri, 12 Aug 2022 17:04:05 +0800
Subject: [PATCH 40/46] Revert "drm/i915: Fix -Wstringop-overflow warning in
 call to intel_read_wm_latency()"

This reverts commit 241b566e0403757b35d251e30798fff6019a6454.
---
 drivers/gpu/drm/i915/intel_pm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index d59455b2d401..e43a76b027a2 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -2822,7 +2822,7 @@ hsw_compute_linetime_wm(const struct intel_crtc_state *crtc_state)
 }
 
 static void intel_read_wm_latency(struct drm_i915_private *dev_priv,
-				  u16 wm[])
+				  u16 wm[8])
 {
 	struct intel_uncore *uncore = &dev_priv->uncore;
 
-- 
2.27.0

